test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -435,3 +483,105 @@ test.requestHooks(mockEnrollAuthenticatorWithCustomApp)('should load select auth
   await t.expect(selectFactorPage.getFactorSelectButtonDataSeByIndex(4, true)).eql('webauthn');
   await t.expect(await selectFactorPage.factorUsageTextExistsByIndex(4)).eql(false);
 });
+
+// re-enable with gen 3 changes
```

Comment:
Is this comment needed?

Reply:
Nope, was just a reminder for us when developing. Removed!